helper/m4/configure.m4

```diff
@@ -17,5 +17,47 @@ AC_ARG_ENABLE([helper-linux],
     [helper_linux=$enableval],
     [helper_linux=yes])
 
+##########################################################################
+# Set optional libcli path
+##########################################################################
+AC_ARG_WITH([libcli-path],
+    [AS_HELP_STRING([--with-libcli-path=DIR],
+        [path to libcli libs and headers [default=system]])],
+    [LIBCLI_CPPFLAGS="-I$withval/include"
+        LIBCLI_LIBS="-L$withval/lib"],
+    [])
+
+##########################################################################
+# Save and set temporary compilation flags
+##########################################################################
+OLD_CPPFLAGS=$CPPFLAGS
+OLD_LIBS=$LIBS
+CPPFLAGS="$LIBCLI_CPPFLAGS $CPPFLAGS"
+LIBS="$LIBCLI_LIBS $LIBS"
+
+#########################################################################
+# If libcli is available, enable CLI helper
+#########################################################################
+helper_cli=no
+AC_CHECK_HEADER(libcli.h,
+    [AC_CHECK_LIB(cli, cli_init, [helper_cli=yes], [])],
+    [])
+
+if test "$helper_cli" = "yes"; then
+    LIBCLI_LIBS="$LIBCLI_LIBS -lcli"
```

Comment:
I think `LIBCLI_LIBS` should be added to Libs.private in `helper/libodphelper.pc.in`. You can check `platform/linux-generic/libodp-linux.pc.in` for examples.